test/evaluators.py

```diff
@@ -9,13 +9,17 @@
 from test.test_utils import evaluate_fuzzy_match
 from test.test_utils import evaluate_must_include
 from test.test_utils import evaluate_ua_match
+from test.test_utils import list_items_in_folder
+from test.test_utils import compress_png
 from typing import Any
 
 from ae.utils.logger import logger
 from playwright.sync_api import CDPSession
 from playwright.sync_api import Page
 from termcolor import colored
 
+import os
+from .validation_agent.validator import validate_task_vqa
```

Comment:
Where is this file? It does not seem to be in the PR

Reply:
The validation_agent directory is another github repo. Is there a best practice for how this repo should be added to this project? Some options: 1) Clone the repo and push 2) Add instructions to cloning and setting up the repo to the Readme

Reply:
The final decision was to add the validation_agent directory independently to this repo.